lib/tasks/grant_superadmin.rake

```diff
@@ -0,0 +1,14 @@
+namespace :superadmin do
+  desc 'Grant the superadmin role to specified users'
+  task 'grant', [:user_list] => [:environment] do |_cmd, args|
+    raise ArgumentError, 'A list of users is required: `rake superadmin:grant[user1,user2,...]`' unless args.user_list
+    args.to_a.each do |u|
```

Comment:
`args` is actually a Rake::TaskArguments object which behaves a bit oddly when multiple arguments are allowed. Any arguments after the first in this task are not named arguments, so they are considered "extras". TaskArgument's .each ignores extras, but to_a captures everything.